protos/farm_ng/core.proto

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) farm-ng, inc.
```

Comment:
@strasdat i wonder if our style guide should be one Message definition per proto file?  It might help us in the long run for knowing where a proto message comes from, and reducing coupling of libraries, and avoiding large recompiles for additions of message types that aren't used elsewhere.